modules/test/playwright/tests/export-import-service/main/remoteStaging.spec.ts

```diff
@@ -45,6 +48,325 @@ export const test = mergeTests(
 	webContentDisplayPageTest
 );
 
+test(
+	'Can publish web content with URL references to live via remote staging',
+	{tag: '@LPS-159626'},
+	async ({
+		apiHelpers,
+		journalEditTemplatePage,
+		page,
+		pageEditorPage,
+		remoteApiHelpers,
+		remotePage,
+		remoteStagingPage,
+		webContentDisplayPage,
+		widgetPagePage
+	}) => {
+		test.slow();
+
+		let featureFlagEnabled;
+		const layouts: Array<Layout> = [];
+		let remoteSite: Site;
+		let remoteUrl: string;
+		let site: Site;	
+
+		try {
+			await test.step('Setup remote staging and pages', async () => {
+				site = await apiHelpers.headlessSite.createSite({
+					name: `site-${getRandomString()}`,
+				});
+
+				apiHelpers.data.push({id: site.id, type: 'site'});
+
+				remoteSite = await remoteApiHelpers.headlessSite.createSite({
+					name: site.name,
+				});
+
+				remoteApiHelpers.data.push({id: remoteSite.id, type: 'site'});
+
+				remoteUrl = remoteApiHelpers.baseUrl.substring(
+					0,
+					remoteApiHelpers.baseUrl.length - 3
+				);
+
+				featureFlagEnabled =
+					await remoteApiHelpers.featureFlag.isFeatureFlagEnabled(
+						'LPD-35914'
+					);
+
+				if (!featureFlagEnabled.featureFlag.enabled) {
+					await remoteApiHelpers.featureFlag.updateFeatureFlag(
+						'LPD-35914',
+						true,
+						true
+					);
+				}
+
+				await apiHelpers.jsonWebServicesStaging.enableRemoteStaging({
+					groupId: site.id,
+					remoteGroupId: remoteSite.id,
+					remotePort,
+				});
+
+				for (const i of [1, 2, 3]) {
+					let layout =
+						await apiHelpers.jsonWebServicesLayout.addLayout({
+							groupId: site.id,
+							title: `Page ${i}`,
+						});
+
+					layouts.push(layout);
+
+					for (const j of [1, 2]) {
+						layout =
+							await apiHelpers.jsonWebServicesLayout.addLayout({
+								groupId: site.id,
+								parentLayoutId: layout.layoutId,
+								title: `Page ${i}${j}`,
+							});
+
+						layouts.push(layout);
+
+						if (i === 1 && j === 1) {
+							layout =
+								await apiHelpers.jsonWebServicesLayout.addLayout(
+									{
+										groupId: site.id,
+										parentLayoutId: layout.layoutId,
+										title: 'Page 111',
+									}
+								);
+							layouts.push(layout);
+						}
+					}
+				}
+			});
```

Comment:
It would be nice if we could extract this to a function (in a page or an util) to create the hierarchy.